apps/test-low-memory.sh

```diff
@@ -1,41 +1,39 @@
 #!/bin/bash
+set -e
 
 # 'npm test' normally does all three of these things.
 # We break them up here so they each run in isolation.
 
-npm run lint && \
-
-node --max_old_space_size=4096 `npm bin`/grunt unitTest && \
-if [ -n "$CIRCLECI" ]; then \
-    bash <(curl -s https://codecov.io/bash) -cF unit; \
-fi && \
-
-node --max_old_space_size=4096 `npm bin`/grunt storybookTest && \
-if [ -n "$CIRCLECI" ]; then \
-    bash <(curl -s https://codecov.io/bash) -cF storybook; \
-fi && \
-
-node --max_old_space_size=4096 `npm bin`/grunt scratchTest && \
-if [ -n "$CIRCLECI" ]; then \
-    bash <(curl -s https://codecov.io/bash) -cF scratch; \
-fi && \
-
-LEVEL_TYPE='turtle' node --max_old_space_size=4096 `npm bin`/grunt integrationTest && \
-if [ -n "$CIRCLECI" ]; then \
-    bash <(curl -s https://codecov.io/bash) -cF integration; \
-fi && \
-
-LEVEL_TYPE='maze|bounce|calc|eval|flappy|studio' node --max_old_space_size=4096 `npm bin`/grunt integrationTest && \
-if [ -n "$CIRCLECI" ]; then \
-    bash <(curl -s https://codecov.io/bash) -cF integration; \
-fi && \
-
-LEVEL_TYPE='applab|gamelab' node --max_old_space_size=4096 `npm bin`/grunt integrationTest && \
-if [ -n "$CIRCLECI" ]; then \
-    bash <(curl -s https://codecov.io/bash) -cF integration; \
-fi && \
-
-LEVEL_TYPE='craft' node --max_old_space_size=4096 `npm bin`/grunt integrationTest && \
-if [ -n "$CIRCLECI" ]; then \
-    bash <(curl -s https://codecov.io/bash) -cF integration; \
+GRUNT_CMD="node --max_old_space_size=4096 `npm bin`/grunt"
+
+if [ -n "$CIRCLECI" ]; then
+  mkdir -p log
+  echo "See apps-test-log under the artifacts tab on circle for test output"
+
+  curl -s https://codecov.io/bash > codecov.sh
+  chmod +x codecov.sh
```

Comment:
Do we need to `.gitignore` codecov.sh?

Reply:
Hmm, I think it makes more sense to just put codecov.sh into `/tmp/` instead of the current directory, so that we don't have to worry about .gitignoring it.